src/ChannelServer/Scripting/ScriptManager.cs

```diff
@@ -413,7 +413,7 @@ private int logdebug(IntPtr L)
 		/// <remarks>
 		/// Parameters:
 		/// - int monsterId
-		/// - string name / Localkey
```

Comment:
Why did you remove this? It documents that you can use either a name or a localization key.